Packs/BmcITSM/ReleaseNotes/1_0_28.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### BMC Helix ITSM
+
+- Fixed an issue where the ***bmc-itsm-task-update*** command would fail when using the **support_company** argument.
```

Comment:
TW recommendation:
Fixed an issue where the ***bmc-itsm-task-update*** command failed when using the *support_company* argument.